doc/api/deprecations.md

```diff
@@ -523,6 +523,17 @@ Type: Runtime
 of V8 5.8. It is replaced by Inspector which is activated with `--inspect`
 instead.
 
+<a id="DEP0063"></a>
+#### DEP0063: ServerResponse.prototype.writeHeader()
```

Comment:
This should have been `###` probably, atm its style is different from all the other deprecation entries on that page.
/cc @jasnell